tests/unit/lms/services/oauth_http_test.py

```diff
@@ -135,6 +136,96 @@ def test_access_token_request_if_theres_no_refresh_token_or_expires_in(
 
         oauth2_token_service.save.assert_called_once_with("access_token", None, None)
 
+    def test_refresh_token_request(
+        self,
+        svc,
+        http_service,
+        OAuthTokenResponseSchema,
+        oauth_token_response_schema,
+        oauth2_token_service,
+    ):
+        oauth_token_response_schema.parse.return_value = {
+            "access_token": "access_token",
+            "refresh_token": "refresh_token",
+            "expires_in": 1234,
+        }
```

Comment:
Feel like I've given this feedback in another PR, but it's probably an even better idea with duplication. I'd suggest putting this in the fixture and referring to the values by reference rather than value.

Reply:
Yep, now that there's two of these I think it makes sense to move it into the fixture. Done: https://github.com/hypothesis/lms/pull/2988/commits/dc51b4d49f843c97783eacc400aaca92c1a6bfe5